euphonic_wrapper.py

```diff
@@ -0,0 +1,194 @@
+import numpy as np
+import os
+from euphonic import Crystal, ForceConstants, QpointPhononModes, DebyeWaller, ureg
+from euphonic.util import mp_grid
+
+class EuphonicWrapper(object):
+    # This a wrapper around the Euphonic ForceConstants and QpointPhononModes classes to make it easier to access from Matlab
+    # It is meant to be used with a Matlab python_wrapper class and implements a horace_sqw function for use with Horace
+
+    defaults = {'force_constants': None, 'phonon_modes': None, 'debye_waller': None, 'debye_waller_grid': None, 
```

Comment:
Do we want to be able to construct the wrapper with a `QpointPhononModes` object? Or just a `ForceConstant`. If just `ForceConstant` then I'll rename the `value` parameter to `force_constant` and make it a required positional_or_keyword argument.

Reply:
I think probably just `ForceConstants` at the moment? To enable simulations with precalculated `QpointPhononModes` the interface would have to look quite different to make sense since the qh, qk, ql arguments wouldn't apply any more... Probably leave that (if there is a need for it) for some point in the future?